### PR TITLE
Update display: contents compat table

### PR DESCRIPTION
**Pull request by gpgekko**
Using the following sources:
 - https://caniuse.com/#feat=css-display-contents
 - https://www.chromestatus.com/feature/5663606012116992
 - https://bugs.webkit.org/show_bug.cgi?id=157477

### CODE REVIEW

File: css/properties/display.json [hunk 1]
```diff
@@ -700,19 +700,25 @@
...
-                  {
-                    "type": "preference",
-                    "name": "Enable experimental Web Platform features"
-                  }
-                ]
+                "version_added": "65"
               },
+              "chrome": [
+                {
+                  "version_added": "58"
```
Comment (teoli2003) [inline css/properties/display.json]: "65" (Enabled by default)

File: css/properties/display.json [hunk 2]
```diff
@@ -700,19 +700,25 @@
...
+                },
+                {
+                  "version_added": true,
```
Comment (teoli2003) [inline css/properties/display.json]: "58" as it was the flag was added in that version.

File: css/properties/display.json [hunk 3]
```diff
@@ -737,22 +743,22 @@
...
-                "version_added": null
+                "version_added": "52"
               },
               "safari": {
-                "version_added": false
+                "version_added": "11.1"
               },
               "safari_ios": {
-                "version_added": null
+                "version_added": "11.1"
```
Comment (teoli2003) [inline css/properties/display.json]: Why 11.1 and not 11.3?

Comment (gpgekko) [reply css/properties/display.json]: 11.3 is the version of iOS that contains the 11.1 version of Safari (as far as I can tell).

Comment (teoli2003) [reply css/properties/display.json]: Makes sense. Thanks.

**Pull request merged**